### PR TITLE
Add Log levels

### PR DESCRIPTION
**Pull request by ibrcko**
A playbook on how to identify what log level to use

### CODE REVIEW

File: playbooks/log-levels.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+description: How to identify what log level to use
+---
+
+Here you can find rules and explanations for when to use which log level.
+
+There are two log level use cases:
+
+## Input/Output information
+
+### LOG (aka. info)
```
Comment (krukru) [inline playbooks/log-levels.md:12]:
```suggestion
### INFO
```

**Comment by krukru** [comment]
I have just found out that there exists a RFC which defines these levels (and is used by PSR-3 Logger Interface specification). What do you think about adopting this PSR/RFC

**Comment by krukru** [comment]
````
0       Emergency: system is unusable
1       Alert: action must be taken immediately
2       Critical: critical conditions
3       Error: error conditions
4       Warning: warning conditions
5       Notice: normal but significant condition
6       Informational: informational messages
7       Debug: debug-level messages
````

**Comment by ibrcko** [comment]
Great catch @krukru!
I doubted that something like this already exists  across the community, but I totally forgot about PSR.

Then I suggest to redo this document in such manner that it will suggest to follow PSR guidelines you mentioned.

What do you think about this suggestion?

**Review by renatoruk — APPROVED** [review]
Looks awesome, a great guideline for structuring logging.

I am not familiar with PSR, but is PSR in some way applicable to JS?

**Comment by krukru** [comment]
PSR is a set of standards for PHP. PSR-3 (you can think of this as standard number 3) proposes an interface for Logger implementations. On the subject of log level, the PSR-3 standard points to an older RFC which is also used in linux OS.
I think this general logic could also be applied to JS, depending on the logger framework used. 
As an example, winston provides some log levels as predefined, but also supports custom log levels (which could map to these 9 levels).
Of course your application does not need to use the entire range of 9 different log levels, but it is nice to have the expressivity to be able to capture or ignore various degrees of log granularity

**Pull request closed without merging**